fabric-network/src/impl/wallet/hsmx509identity.ts

```diff
@@ -83,21 +86,21 @@ export class HsmX509Provider implements IdentityProvider {
 		const data: HsmX509IdentityDataV1 = {
 			credentials: {
 				certificate: identity.credentials.certificate,
+				privateKey: identity.credentials.privateKey
 			},
 			mspId: identity.mspId,
 			type: 'HSM-X.509',
-			version: 1,
+			version: 2,
```

Comment:
Perfect except (as above) the interface type should be `HsmX509IdentityDataV2` for v2 data